packages/govuk-frontend/tasks/build/package.mjs

```diff
@@ -16,19 +16,12 @@ export default (options) => gulp.series(
     files.clean('*', options)
   ),
 
+  assets(options),
```

Comment:
Should we share this new task with `release.mjs` too?

It's mostly identical between `package.mjs` and `release.mjs`

Would just need `destPath` overriding without the `govuk/` prefix

Reply:
Will leave this as a future improvement if that's alright.